components/Login.css

```diff
@@ -0,0 +1,86 @@
+.wrapper {
+  max-width: 40rem;
+  margin-top: 8rem;
+  margin-bottom: 2rem;
+  margin-left: 60rem;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  padding: 3rem;
+  border-radius: 0.5rem;
+  box-shadow: 0 5px 10px rgba(0, 0, 0, 0.2);
+}
+
+.login-wrapper {
+  margin-top: 20rem;
+  margin-bottom: 6rem;
+}
+.wrapper h2 {
+  position: relative;
+  font-size: 4rem;
+  font-weight: 600;
+  color: var(--primary-dark-color);
+}
+
+.wrapper form {
+  margin-top: 1rem;
+}
+
+.wrapper form .input-box {
+  height: 3.5rem;
+  margin: 1.2rem 0;
+}
+
+form .input-box input {
+  height: 100%;
+  width: 100%;
+  outline: none;
+  padding: 0 1rem;
+  font-size: 1.6rem;
+  font-weight: 400;
+  color: var(--text-white);
```

Comment:
Here you set the text color to white inside the input which matches your background. So, change that.